### PR TITLE
Updating information about emails

### PR DESCRIPTION
**Pull request by yabellini**

### CODE REVIEW

File: content/organization/tech/accounts/index.en.md [hunk 1]
```diff
@@ -83,7 +83,6 @@ A good strategy might be to make your group private and ask candidates questions
 ## E-Mail
 
 * You can get a chapter email address (`yourcity@rladies.org`) which can be set up to forward to all the organisers of a chapter.  As of March 18, 2019, we are no longer issuing personal rladies.org emails (`yourname@rladies.org`) to organizers and will have chapter emails only.
```
Comment (yabellini) [inline content/organization/tech/accounts/index.en.md:85]:
```suggestion
* You can get a chapter email address (`yourcity@rladies.org`).  As of March 18, 2019, we are no longer issuing personal rladies.org emails (`yourname@rladies.org`) to chapters organizers and will have chapter emails only. 
```

File: content/organization/tech/accounts/index.en.md [hunk 2]
```diff
@@ -82,8 +82,7 @@ A good strategy might be to make your group private and ask candidates questions
...
 
-* You can get a chapter email address (`yourcity@rladies.org`) which can be set up to forward to all the organisers of a chapter.  As of March 18, 2019, we are no longer issuing personal rladies.org emails (`yourname@rladies.org`) to organizers and will have chapter emails only. 
-* Starting in 2021, new chapter emails are just forwarding addresses[^contextemail]. With a forwarding email address, there’s no inbox or password, just a public facing email address that forwards to your local organiser team. 
+* You can get a chapter email address (`yourcity@rladies.org`).  As of March 18, 2019, we are no longer issuing personal rladies.org emails (`yourname@rladies.org`) to chapters organizers and will have chapter emails only. 
 * Put the chapter email in cc when answering an email so that your co-organizers can see the answer.
 * The idea with the chapter email address is that you can use this as a public facing email address for your community, so members or meetup hosts (a company or university) can get in touch with you easily without you having to list a personal email address publicly. Then you can use your personal email address to reply. 
 * To request a new chapter email:
```
Comment (yabellini) [inline content/organization/tech/accounts/index.en.md:88]:
```suggestion
* Your chapter will get a new email during the onboarding process. You don't need to do any request. You will get an email from R-Ladies Google Workspace with the amail adress, the temporary password, the link to login to your account for the first time and change the password and instruction about to create save and strong passwords. 
```

File: content/organization/tech/accounts/index.en.md [hunk 3]
```diff
@@ -82,11 +82,10 @@ A good strategy might be to make your group private and ask candidates questions
...
 * Put the chapter email in cc when answering an email so that your co-organizers can see the answer.
 * The idea with the chapter email address is that you can use this as a public facing email address for your community, so members or meetup hosts (a company or university) can get in touch with you easily without you having to list a personal email address publicly. Then you can use your personal email address to reply. 
-* To request a new chapter email:
+* Your chapter will get a new email during the onboarding process. You don't need to do any request. You will get an email from R-Ladies Google Workspace with the amail adress, the temporary password, the link to login to your account for the first time and change the password and instruction about to create save and strong passwords. 
    * Send an email (`email@rladies.org`) and a volunteer will set up your chapter email for you.  Please be patient, as these requests are currently being processed on a monthly basis.
```
Comment (yabellini) [inline content/organization/tech/accounts/index.en.md:89]:
```suggestion

```

File: content/organization/tech/accounts/index.en.md [hunk 4]
```diff
@@ -82,12 +82,10 @@ A good strategy might be to make your group private and ask candidates questions
...
 * Put the chapter email in cc when answering an email so that your co-organizers can see the answer.
 * The idea with the chapter email address is that you can use this as a public facing email address for your community, so members or meetup hosts (a company or university) can get in touch with you easily without you having to list a personal email address publicly. Then you can use your personal email address to reply. 
-* To request a new chapter email:
-   * Send an email (`email@rladies.org`) and a volunteer will set up your chapter email for you.  Please be patient, as these requests are currently being processed on a monthly basis.
+* Your chapter will get a new email during the onboarding process. You don't need to do any request. You will get an email from R-Ladies Google Workspace with the amail adress, the temporary password, the link to login to your account for the first time and change the password and instruction about to create save and strong passwords. 
    * In the email, please send the personal email addresses for you and your co-organisers if you would like chapter email to forward there.
```
Comment (yabellini) [inline content/organization/tech/accounts/index.en.md:89]:
```suggestion

```

File: content/organization/tech/accounts/index.en.md [hunk 5]
```diff
@@ -82,13 +82,10 @@ A good strategy might be to make your group private and ask candidates questions
...
 * Put the chapter email in cc when answering an email so that your co-organizers can see the answer.
 * The idea with the chapter email address is that you can use this as a public facing email address for your community, so members or meetup hosts (a company or university) can get in touch with you easily without you having to list a personal email address publicly. Then you can use your personal email address to reply. 
-* To request a new chapter email:
-   * Send an email (`email@rladies.org`) and a volunteer will set up your chapter email for you.  Please be patient, as these requests are currently being processed on a monthly basis.
-   * In the email, please send the personal email addresses for you and your co-organisers if you would like chapter email to forward there.
+* Your chapter will get a new email during the onboarding process. You don't need to do any request. You will get an email from R-Ladies Google Workspace with the amail adress, the temporary password, the link to login to your account for the first time and change the password and instruction about to create save and strong passwords. 
    
 ```
```
Comment (yabellini) [inline content/organization/tech/accounts/index.en.md:90]:
```suggestion

```

File: content/organization/tech/accounts/index.en.md [hunk 6]
```diff
@@ -82,13 +82,10 @@ A good strategy might be to make your group private and ask candidates questions
...
 Dear R-Ladies global team,
```
Comment (yabellini) [inline content/organization/tech/accounts/index.en.md:91]:
```suggestion

```

**Review by yabellini — COMMENTED** [review]
removing email text, because they shouldn't sent any email requesting emails :-)

File: content/organization/tech/accounts/index.en.md [hunk 7]
```diff
@@ -82,16 +82,11 @@ A good strategy might be to make your group private and ask candidates questions
...
 * The idea with the chapter email address is that you can use this as a public facing email address for your community, so members or meetup hosts (a company or university) can get in touch with you easily without you having to list a personal email address publicly. Then you can use your personal email address to reply. 
-* To request a new chapter email:
-   * Send an email (`email@rladies.org`) and a volunteer will set up your chapter email for you.  Please be patient, as these requests are currently being processed on a monthly basis.
-   * In the email, please send the personal email addresses for you and your co-organisers if you would like chapter email to forward there.
+* Your chapter will get a new email during the onboarding process. You don't need to do any request. You will get an email from R-Ladies Google Workspace with the amail adress, the temporary password, the link to login to your account for the first time and change the password and instruction about to create save and strong passwords. 
    
-```
-Dear R-Ladies global team,
 
 I'd like to request an email for the chapter R-Ladies Pawnee.
```
Comment (yabellini) [inline content/organization/tech/accounts/index.en.md:91]:
```suggestion

```

File: content/organization/tech/accounts/index.en.md [hunk 8]
```diff
@@ -82,18 +82,12 @@ A good strategy might be to make your group private and ask candidates questions
...
-   * Send an email (`email@rladies.org`) and a volunteer will set up your chapter email for you.  Please be patient, as these requests are currently being processed on a monthly basis.
-   * In the email, please send the personal email addresses for you and your co-organisers if you would like chapter email to forward there.
+* Your chapter will get a new email during the onboarding process. You don't need to do any request. You will get an email from R-Ladies Google Workspace with the amail adress, the temporary password, the link to login to your account for the first time and change the password and instruction about to create save and strong passwords. 
    
-```
-Dear R-Ladies global team,
 
-I'd like to request an email for the chapter R-Ladies Pawnee.
 
 Here are the addresses that the email should forward to:
```
Comment (yabellini) [inline content/organization/tech/accounts/index.en.md:92]:
```suggestion

```

File: content/organization/tech/accounts/index.en.md [hunk 9]
```diff
@@ -82,20 +82,13 @@ A good strategy might be to make your group private and ask candidates questions
...
+* Your chapter will get a new email during the onboarding process. You don't need to do any request. You will get an email from R-Ladies Google Workspace with the amail adress, the temporary password, the link to login to your account for the first time and change the password and instruction about to create save and strong passwords. 
    
-```
-Dear R-Ladies global team,
 
-I'd like to request an email for the chapter R-Ladies Pawnee.
 
-Here are the addresses that the email should forward to:
 
 - Leslie Knope (leslie@knope.ok),
```
Comment (yabellini) [inline content/organization/tech/accounts/index.en.md:93]:
```suggestion

```

File: content/organization/tech/accounts/index.en.md [hunk 10]
```diff
@@ -82,20 +82,13 @@ A good strategy might be to make your group private and ask candidates questions
...
 - April Ludgate (april@ludgate.ok).
```
Comment (yabellini) [inline content/organization/tech/accounts/index.en.md:94]:
```suggestion

```

File: content/organization/tech/accounts/index.en.md [hunk 11]
```diff
@@ -82,23 +82,14 @@ A good strategy might be to make your group private and ask candidates questions
...
-Dear R-Ladies global team,
 
-I'd like to request an email for the chapter R-Ladies Pawnee.
 
-Here are the addresses that the email should forward to:
 
-- Leslie Knope (leslie@knope.ok),
-- April Ludgate (april@ludgate.ok).
 
 Thank you, best wishes
```
Comment (yabellini) [inline content/organization/tech/accounts/index.en.md:94]:
```suggestion

```

File: content/organization/tech/accounts/index.en.md [hunk 12]
```diff
@@ -82,25 +82,15 @@ A good strategy might be to make your group private and ask candidates questions
...
 
-I'd like to request an email for the chapter R-Ladies Pawnee.
 
-Here are the addresses that the email should forward to:
 
-- Leslie Knope (leslie@knope.ok),
-- April Ludgate (april@ludgate.ok).
 
-Thank you, best wishes
 ```
```
Comment (yabellini) [inline content/organization/tech/accounts/index.en.md:94]:
```suggestion

```

File: content/organization/tech/accounts/index.en.md [hunk 13]
```diff
@@ -82,26 +82,15 @@ A good strategy might be to make your group private and ask candidates questions
...
 
-Here are the addresses that the email should forward to:
 
-- Leslie Knope (leslie@knope.ok),
-- April Ludgate (april@ludgate.ok).
 
-Thank you, best wishes
-```
    
 * If you add new organizers to your team, please send an email to `email@rladies.org` to request that their personal email be added to the chapter email forwarding.
```
Comment (yabellini) [inline content/organization/tech/accounts/index.en.md:95]:
```suggestion

```

File: content/organization/tech/accounts/index.en.md [hunk 14]
```diff
@@ -82,28 +82,16 @@ A good strategy might be to make your group private and ask candidates questions
...
 
-- Leslie Knope (leslie@knope.ok),
-- April Ludgate (april@ludgate.ok).
 
-Thank you, best wishes
-```
    
-* If you add new organizers to your team, please send an email to `email@rladies.org` to request that their personal email be added to the chapter email forwarding.  
 
 ```
```
Comment (yabellini) [inline content/organization/tech/accounts/index.en.md:96]:
```suggestion

```

File: content/organization/tech/accounts/index.en.md [hunk 15]
```diff
@@ -82,28 +82,16 @@ A good strategy might be to make your group private and ask candidates questions
...
 Dear R-Ladies global team,
```
Comment (yabellini) [inline content/organization/tech/accounts/index.en.md:97]:
```suggestion

```

File: content/organization/tech/accounts/index.en.md [hunk 16]
```diff
@@ -82,31 +82,17 @@ A good strategy might be to make your group private and ask candidates questions
...
 
-Thank you, best wishes
-```
    
-* If you add new organizers to your team, please send an email to `email@rladies.org` to request that their personal email be added to the chapter email forwarding.  
 
-```
-Dear R-Ladies global team,
 
 I'd like to request adding an email to the R-Ladies Pawnee chapter email forwarding: our new co-organizer Leslie Knope (leslie@knope.ok).
```
Comment (yabellini) [inline content/organization/tech/accounts/index.en.md:97]:
```suggestion

```

File: content/organization/tech/accounts/index.en.md [hunk 17]
```diff
@@ -82,33 +82,18 @@ A good strategy might be to make your group private and ask candidates questions
...
-```
    
-* If you add new organizers to your team, please send an email to `email@rladies.org` to request that their personal email be added to the chapter email forwarding.  
 
-```
-Dear R-Ladies global team,
 
-I'd like to request adding an email to the R-Ladies Pawnee chapter email forwarding: our new co-organizer Leslie Knope (leslie@knope.ok).
 
 Thank you, best wishes
```
Comment (yabellini) [inline content/organization/tech/accounts/index.en.md:98]:
```suggestion

```

File: content/organization/tech/accounts/index.en.md [hunk 18]
```diff
@@ -82,33 +82,18 @@ A good strategy might be to make your group private and ask candidates questions
...
 ```
```
Comment (yabellini) [inline content/organization/tech/accounts/index.en.md:99]:
```suggestion

```

File: content/organization/tech/accounts/index.en.md [hunk 19]
```diff
@@ -82,47 +82,23 @@ A good strategy might be to make your group private and ask candidates questions
...
-* Put the chapter email in cc when answering an email so that your co-organizers can see the answer.
-* The idea with the chapter email address is that you can use this as a public facing email address for your community, so members or meetup hosts (a company or university) can get in touch with you easily without you having to list a personal email address publicly. Then you can use your personal email address to reply. 
-* To request a new chapter email:
-   * Send an email (`email@rladies.org`) and a volunteer will set up your chapter email for you.  Please be patient, as these requests are currently being processed on a monthly basis.
-   * In the email, please send the personal email addresses for you and your co-organisers if you would like chapter email to forward there.
-   
-```
-Dear R-Ladies global team,
+As of November 2022 R-Ladies email use a Google Workspace to manage emails accounts.  We provide chapters email address (`yourcity@rladies.org`) during the onboarding process. 
+_You don't need to do any request_. After the creation of your chapter account you will get an email from R-Ladies Google Workspace with the amail
```
Comment (drmowinckels) [inline content/organization/tech/accounts/index.en.md:86]:
```suggestion
_You do not need to request an email_, after the creation of your chapter account you will get an email from R-Ladies Google Workspace with information about your chapter email.
```

File: content/organization/tech/accounts/index.en.md [hunk 20]
```diff
@@ -82,47 +82,23 @@ A good strategy might be to make your group private and ask candidates questions
...
-* If you add new organizers to your team, please send an email to `email@rladies.org` to request that their personal email be added to the chapter email forwarding.  
-
-```
-Dear R-Ladies global team,
-
-I'd like to request adding an email to the R-Ladies Pawnee chapter email forwarding: our new co-organizer Leslie Knope (leslie@knope.ok).
-
-Thank you, best wishes
-```
+The idea with the chapter email address is that you can use this as a public facing email address for your community,
```
Comment (drmowinckels) [inline content/organization/tech/accounts/index.en.md:89]:
```suggestion
The chapter email address is the public facing email address for your community, 
```

File: content/organization/tech/accounts/index.en.md [hunk 21]
```diff
@@ -82,47 +82,23 @@ A good strategy might be to make your group private and ask candidates questions
...
-
-```
-Dear R-Ladies global team,
-
-I'd like to request adding an email to the R-Ladies Pawnee chapter email forwarding: our new co-organizer Leslie Knope (leslie@knope.ok).
-
-Thank you, best wishes
-```
+The idea with the chapter email address is that you can use this as a public facing email address for your community, 
+so members or meetup hosts (a company or university) can get in touch with you easily without you having to list a personal
```
Comment (drmowinckels) [inline content/organization/tech/accounts/index.en.md:90]:
```suggestion
so members or meetup hosts (a company or university) can get in touch with you and your co-organisers easily.
```

File: content/organization/tech/accounts/index.en.md [hunk 22]
```diff
@@ -82,47 +82,23 @@ A good strategy might be to make your group private and ask candidates questions
...
-```
-Dear R-Ladies global team,
-
-I'd like to request adding an email to the R-Ladies Pawnee chapter email forwarding: our new co-organizer Leslie Knope (leslie@knope.ok).
-
-Thank you, best wishes
-```
+The idea with the chapter email address is that you can use this as a public facing email address for your community, 
+so members or meetup hosts (a company or university) can get in touch with you easily without you having to list a personal 
+email address publicly. Then you can use your personal email address to reply.  We recommend you put the chapter email in cc when
```
Comment (drmowinckels) [inline content/organization/tech/accounts/index.en.md:91]:
```suggestion
You can then use your personal email address to reply., but we recommend you put the chapter email in cc when 
```

**Review by drmowinckels — CHANGES_REQUESTED** [review]

**Comment by yabellini** [comment]
I think this is ready to merge, so we have the email information updated on the guide.  Thanks @drmowinckels for reviewing my changes.

**Pull request merged**